### DISCUSSION TITLE
Share note

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#338] [Ideas]
Hi

It would be great if one could share notes through the server. It could be simple wbe page with the note in there.

thanks

**Comment by vrtmrz** [comment]
I am really sorry for being late!

I would love to introduce this!
[LiveSync Subscribe Publish Kit](https://github.com/vrtmrz/livesync-subscribe-publish-kit) for serving blog.
This consists of two entities. Hugo for building the server, and [LiveSync Subscribe Publish](https://github.com/vrtmrz/livesync-subscribe-publish) for receiving files. 

If you want to do something more fine-grained, [LiveSync Bridge](https://github.com/vrtmrz/livesync-bridge) could be useful.

I hope that these sub-projects help you.

**Reply by gerroon** [reply]
Thanks, but this is way too complicated? It seems to be for publishing the whole thing. I was looking for a way to publish a single note as a simple page for instance. I do not want to publish my whole notes, just a single note with a secret url occasionally.

**Reply by vrtmrz** [reply]
Now I understand. Thank you for your patience in explaining it to me!

Actually, we lack it, but it would certainly be nice to have something like that indeed.

**Reply by hilsonp** [reply]
Do you sometime sleep ??? This is awesome ;-)

**Comment by batuozdemir** [comment]
I came here to this! This is an amazing plugin, and it seems to me the livesync functionality would be great for sharing notes with my coworkers or friends. 

I guess this would need its own plugin because it's use case is different enough, but what I have in mind is:

-I have couchdb setup on, let's say, the _obsidian-liveshare_ plugin (the same couchdb that I use with obsidian-livesync plugin)
-I can share notes with other people who have the same plugin installed
-I give read/write privileges specific to the note I shared to the people I share it with
-They can read/edit the note and I can see the changes on my obsidian vault.
-This can also be used to share folders.

**Comment by cwsqer006** [comment]
cvfbghnjm
